setup.py

```diff
@@ -17,7 +21,7 @@
     author_email="visidata@saul.pw",
     url="https://visidata.org",
     download_url="https://github.com/saulpw/visidata/tarball/" + __version__,
-    scripts=["bin/vd", "bin/vd2to3.vdx"],
+    scripts=[vd_bin, "bin/vd2to3.vdx"],
     entry_points={
         "console_scripts": ["visidata=visidata.main:vd_cli"],
```

Comment:
I've been doing some more research.
Is there a reason we don't just make the following change?

```suggestion
        "console_scripts": ["vd=visidata.main:vd_cli"],
```

This will automatically create `vd.exe` on Windows, and `vd` on everything else.
We can then remove `vd_bin` from `scripts` entirely.

Additionally, I'm not sure what `vd2to3.vdx` does, but it may also make sense to add it to `entry_points["console_scripts"]` too, to ensure it automatically generates a Windows compatible executable wrapper as required.

Reply:
I think it's likely wise to keep both `vd` and `visidata` just so you don't break anything that may assume the full name. The only negative of the two executables is size. `vd.cmd` is 11 bytes. `vd.exe` would be 108,393 bytes. 108K isn't a big deal, though I'm old enough to have had floppy disks with less capacity. :D

Reply:
Excellent suggestion @daviewales!

@ptyork I just pushed an update with this suggestion. Could you install it and see if things still work on Windows?